### ISSUE TITLE
[Functional-Social Embedded-Create Profile]: No textual information is provided for “Required” input fields

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#8]
"Priority: 

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Verify that there is any information provided regarding “Required” fields.

Expected
Textual information should be provided for the “Text” boxes, so that screen reader user or keyboard user will get proper information regarding “Required” fields.

Actual
There is no textual information is provided for “Required” fields, And screen reader also does not convey anything regarding “required” fields. While “First Name” and “Last Name” text box is required fields.

MAS Rule: Functional

Impact
User who is rely on screen reader or keyboard will not get any information if there is no textual information is provided for “Required” fields.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
